tests/test_timeouts.py

```diff
@@ -12,6 +14,9 @@ async def test_read_timeout(server):
             await client.get(server.url.copy_with(path="/slow_response"))
 
 
+@pytest.mark.skipif(
+    sys.platform == "win32", reason="time related tests are flaky on win32"
+)
```

Comment:
I assume we could drop this, and this particular test *would actually* pass okay on windows?

(Also, I think we might want to raise a related refactoring issue to remove these `/slow_response` tests completely.)

Reply:
this test **may** pass on windows.

Reply:
it's a time accuracy on windows. we could increate sleep time in `/slow_response` to several seconds to fix this.